rabbitmq.py

```diff
@@ -250,7 +250,9 @@ def config(config_values):
     b = Broker()
 
     # We use only single values so this works fine
-    config_map = {c.key: c.values[0] for c in config_values.children}
+    config_map = {}
+    for child in config_values.children:
+        config_map[child.key] = child.values[0]
```

Comment:
More pythonic way is:
`config_map = dict([(c.key, c.values[0]) for c in config_values.children])`

but you code works too.